stages/org.osbuild.ignition

```diff
@@ -22,6 +22,7 @@ import sys
 import osbuild.api
 
 STAGE_OPTS = """
+"additionalProperties": false,
```

Comment:
The schema should be called `SCHEMA`, not `STAGE_OPTS`.
The schema is basically not read here.

Reply:
Was testing the stage and tried to add additional props to make it break and it wasn't failing.
That seems to be an old thing, changed in 2a9cdde5ec8392f3f70ca88d751b6cbe63f7386f, and it somehow survived the move.

Reply:
oh, must've been a PR for the ignition stage open while the move was happening but was merged after.  Funny.

Reply:
I was actually wondering about this one and thought it is weird, but didn't spend time on investigating this further. It is interesting to see this additional information 🤔